downstream/assemblies/platform/assembly-using-rhsso-operator-with-automation-hub.adoc

```diff
@@ -0,0 +1,37 @@
+ifdef::context[:parent-context: {context}]
+
+[id="using-rhsso-operator-with-automation-hub"]
+
+:context: using-a-rhsso-operator
+
+= Using a Red Hat Single Sign On Operator with {HubName}
+
+The _{PlatformName}_ (AAP) _{PrivateHubNameStart}_ (PAH) uses Red Hat Single Sign-On (RH-SSO) for external authentication.
+
+The Red Hat Single Sign-On Operator automates Red Hat Single Sign-On administration in Openshift.
+Use this Operator to create custom resources, which automate administrative tasks.
```

Comment:
I think there's some nuances missing here. Operators create and manage custom resources in order to automate administrative tasks. This wording makes it sound like the CRs automate administrative tasks but CRs are simply objects in k8s.